var/spack/repos/builtin/packages/ace/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os import environ as env
+from spack import *
+
+
+class Ace(Package):
+    """ACE is an open-source framework that provides many components and
+       patterns for developing high-performance, distributed real-time and
+       embedded systems. ACE provides powerful, yet efficient abstractions
+       for sockets, demultiplexing loops, threads, synchronization
+       primitives."""
+
+    homepage = "http://www.dre.vanderbilt.edu/~schmidt/ACE.html"
+    url = "http://download.dre.vanderbilt.edu/previous_versions/" \
+          "ACE-6.5.1.tar.gz"
+
+    version('6.5.6', '7717cad84d4a9c3d6b2c47963eb555d96de0be657870bcab6fcef4'
+                     'c0423af0de')
+    version('6.5.1', '1f318adadb19da23c9be570a9c600a330056b18950fe0bf0eb1cf5'
+                     'cac8b72a32')
+    version('6.5.0', 'b6f9ec922fbdcecb4348e16d851d0d1f135df1836dfe77d2e0b642'
+                     '95ddb83066')
```

Comment:
You can put this all on one line, Spack exempts checksums from the line length checks.